cmd/wksctl/profile/enable/enable.go

```diff
@@ -0,0 +1,119 @@
+package enable
```

Comment:
Please apply all comments from `package disable` to this file as well. :)

Reply:
done